src/main/java/world/bentobox/bentobox/listeners/flags/protection/HurtingListener.java

```diff
@@ -53,7 +52,7 @@ public void onEntityDamage(final EntityDamageByEntityEvent e) {
         // Mobs being hurt
         if (Util.isPassiveEntity(e.getEntity())) {
             respond(e, e.getDamager(), Flags.HURT_ANIMALS);
-        } else if (e.getEntity() instanceof AbstractVillager) {
+        } else if (e.getEntity().getType() == EntityType.VILLAGER) {
```

Comment:
This change is not correct as it still requires a wandering trader.
That's why it was changed to the abstract villager.